tmail-backend/deployment-tests/distributed-es6-backport/src/test/java/com/linagora/tmail/deployment/TmailDistributedEs6Extension.java

```diff
@@ -61,11 +61,6 @@ public void beforeEach(ExtensionContext extensionContext) throws IOException {
             Paths.get("tmail-backend", "apps", "distributed-es6-backport", "target", "jib-image.tar").toString());
         james.getDockerClient().loadImageCmd(Files.newInputStream(Paths.get(dockerSaveFileUrl))).exec();
 
-        Runnables.runParallel(
-            cassandra::start,
-            elasticsearch::start,
-            rabbitmq::start,
-            s3::start);
```

Comment:
You don't start those containers anymore? Did I miss something? :)

Reply:
Testcontainer library will start it 
because already the declare `.dependsOn(cassandra, opensearch, s3, rabbitmq, ldap`...